CHANGES.rst

```diff
@@ -19,6 +19,7 @@ astropy.coordinates
 - Numpy functions that broadcast, change shape, or index (like ``np.broadcast_to``,
   ``np.rot90``, or ``np.roll``) now work on coordinates, frames, and
   representations. [#10337]
+- ``galactocentric_frame_defaults`` registry for parameter values and metadata.
```

Comment:
This needs a PR number added to it to get rid of the bot failure. I'd also expand the sentence a bit; maybe simply:
```
``galactocentric_frame_defaults`` can now also be used as a registry, with user-defined parameter values and metadata. [#10624]
```
I thought there normally also was an empty line between entries, which used to be required for sphinx to typeset correctly, but perhaps that doesn't matter any more.

Reply:
- [ ]  Added number
- [ ]  used suggested sentence
- [ ]  added line